docs/designers-developers/developers/tutorials/format-api/1-register-format.md

```diff
@@ -0,0 +1,49 @@
+# Register a new format
+
+The first thing you're going to do in this tutorial is to register the new format that the plugin intends to apply. WordPress has the [`registerFormatType`](/packages/rich-text/README.md#registerFormatType) primitive to do so.
+
+Let's prepare a minimal plugin to make this work. Create a new folder and a file named `my-custom-format.php` within it containing the necessary PHP code to register and enqueue the JavaScript assets:
+
+```php
+<?php
+
```

Comment:
Super minor but in the other tutorials we don't have a newline between the plugin doc block and opening PHP tag. Not a block to merge, just a note for the future :)

Reply:
04bd2e50d44edc7fee4040eed58517afc230d8b6 removes the space